fedimint-core/src/module/mod.rs

```diff
@@ -186,11 +205,10 @@ macro_rules! __api_endpoint {
 
             async fn handle<'a, 'b>(
                 $state: &'a Self::State,
-                $dbtx: &'a mut fedimint_core::db::ModuleDatabaseTransaction<'b, ModuleInstanceId>,
+                $context: &'a mut fedimint_core::module::ApiEndpointContext<'b>,
```

Comment:
not introduced by this PR, but this should be `$crate::module::ApiEndpointContext`

Reply:
Why is that?

Reply:
$crate:: is more correct than fedimint_core, it even works in case fedimint_core is not directly imported by the crate using the macro (reexported by some other crate)